src/Raven.Server/Config/Categories/StorageConfiguration.cs

```diff
@@ -158,5 +158,10 @@ public StorageConfiguration()
         [MinValue(0)]
         [ConfigurationEntry("Storage.MaxNumberOfRecyclableJournals", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public int MaxNumberOfRecyclableJournals { get; set; }
+
+        [Description("Disable further usage of sparse regions for the FS to reclaim free pages. In order to clear existing sparse regions you need to do that manually.")]
```

Comment:
That is a great question, which I don't the answer for it.

I think we will need to create an dedicated EP, or figure out the proper linux/win command and document it later on.

Reply:
@ayende Thoughts?

Reply:
On Linux, the command is something like:


```
FILENAME=/path/to/Raven.voron
$ fallocate -l $(stat -c%s $FILENAME) $FILENAME
```

Windows (requires admin rights):

```
$fileSize = (Get-Item $Filename).Length
fsutil file setValidData $Filename $fileSize
```

Note that we don't generally recommend that, and let the OS fill it it on as needed basis.

Reply:
I'm for documenting native OS commands (instead of providing an endpoint for that).

Reply:
FYI that for windows the command is `fsutil sparse setflag $Filename 0`
for linux I will check once this will merged and deploy this on our test cloud